packages/base/Accordion/src/Accordion/Accordion.tsx

```diff
@@ -110,6 +114,22 @@ export const Accordion = forwardRef<HTMLElement, Props>(function Accordion(
 
   const appliedBorders = children || expanded ? (borders as Borders) : 'none'
 
+  const renderExpandIcon = (expandIconPosition: 'left' | 'right') => {
+    if (expandIcon) {
+      return decorateWithExpandIconClasses(expandIcon, expandIconClass)
+    }
+
+    return (
+      <div
+        className={`${classes.expandIconAlignTop} ${
```

Comment:
Add this first:
`import cx from 'classnames'`

```suggestion
        className={cx({
           [classes.expandIconAlignTop]: true,
           [classes.expandIconLeft]: expandIconPosition === 'left'
           })}
```